src/common/snippets/tests/src/pass/mlp_tokenization.cpp

```diff
@@ -0,0 +1,110 @@
+// Copyright (C) 2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <gtest/gtest.h>
+
+#include <pass/mlp_tokenization.hpp>
+#include <subgraph_mlp.hpp>
+
+#include "openvino/pass/serialize.hpp"
```

Comment:
Minor:
I believe this include is extra. Remove it please

Reply:
Done